src/sfdt/sfdtHelpers/paragraphHelpers.ts

```diff
@@ -0,0 +1,58 @@
+import get from 'lodash/get';
+import merge from 'lodash/merge';
+import {baseStyles} from './sfdtConstants';
+import {block as BlockType, style as StyleType, paragraphFormat as ParagraphFormatType} from '../../../types/sfdt';
+/**
+ * Returns style object with its name (unique) property as given name. If not found, returns undefined
+ * @param name style name
+ * @param styles styles array from sfdt
+ */
+export const getStyleFromName = (name: string, styles: StyleType[] = []) => {
+	return styles.filter((style) => style.name === name)[0];
+};
+
+/**
+ * Returns the updated paragraphFormat from styles format
+ * @param name style name
+ * @param stylesArray sfdt styles array
+ * @param currentParaFormat running paraFormat in each iteration updated with the given style paraFormat
+ */
+const parseParaStyle = (name: string, stylesArray: StyleType[], currentParaFormat: ParagraphFormatType) => {
+	const paraStyle = getStyleFromName(name, stylesArray);
```

Comment:
There's always a chance that `paraStyle` might be undefined (ie if filter doesnt match anything), so please add a guard.

ie:
`if (paraStyle  && ....) {}`